terraform/database.tf

```diff
@@ -233,14 +233,18 @@ resource "null_resource" "migrate" {
       DB_CONN                 = google_sql_database_instance.db-inst.connection_name
       DB_ENCRYPTION_KEY       = google_kms_crypto_key.database-encrypter.self_link
       DB_NAME                 = google_sql_database.db.name
-      DB_PASSWORD             = google_secret_manager_secret_version.db-secret-version["password"].name
+      DB_PASSWORD             = "secret://${google_secret_manager_secret_version.db-secret-version["password"].id}"
```

Comment:
Hmmmmmm.... this change is correct, but how did it work before?

Reply:
That was puzzling me too. But it was late and I was all shrugs. Maybe no one has run this step in a long time?